index.md

```diff
@@ -5,7 +5,7 @@ permalink: /
 ---
 
 
-**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents.
+**JSON Schema** is a declarative domain specific language that allows you to **annotate** and **validate** JSON documents. JSON Schema enables the confident and reliable use of the JSON data format.
```

Comment:
From the discussion on the old PR, it looks like @awwright and @jdesrosiers agreed that this should be change to just "declarative language", which I also prefer.

Reply:
Either version works for me, but I think leaving out "domain specific" is more accessible and it effectively says the same thing either way.

Reply:
Yeah.  Even though I know what "domain specific" means it still threw me a bit trying to figure out exactly how it was meant here.